hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.tsx

```diff
@@ -230,18 +230,29 @@ export const DetailInfoSection = ({
                 Manufacturer
             </Typography>
             <Typography>{manufacturer}</Typography>
-            <Typography variant="body2" className={styles.heading}>
+            <Typography
```

Comment:
I think it would be better to take advantage of jsx here
```
{
   modelNumber && <>
      <Typography> ... </Typography>
      <Typography> ... </Typography>
   </>
}
```
same for the datasheet prop

Reply:
Sounds good! Just made the change. Thanks YuYing!